frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx

```diff
@@ -33,20 +33,28 @@ interface DashboardInfoSidebarProps {
   dashboard: Dashboard;
   setDashboardAttribute: (name: string, value: DashboardAttributeType) => void;
   saveDashboardAndCards: (preserveParameters?: boolean) => void;
-  revisions: RevisionType[];
-  currentUser: User;
-  revertToRevision: (revision: RevisionType) => void;
+  revisions: RevisionType[] | undefined;
 }
 
-const DashboardInfoSidebar = ({
+export const DashboardInfoSidebar = _.compose(
+  Revision.loadList({
```

Comment:
Do we want to try building a `use-revision-list` hook rather than wrapping the component?